crates/standalone/src/routes/mod.rs

```diff
@@ -7,14 +7,17 @@ use spacetimedb_client_api::{
 };
 
 #[allow(clippy::let_and_return)]
-pub fn router<S>() -> axum::Router<S>
+pub fn router<S>(ctx: S) -> axum::Router<()>
```

Comment:
I think this (and maybe the new parameters to other similar functions) cause breakage in `SpacetimeDBPrivate`